src/components/shared/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactNode } from 'react';
+import { FormattedDateParts } from 'react-intl';
+
+import Tag, { Appearance as TagAppearance } from '~shared/Tag';
+import { Message } from '~types';
+import { MotionState, MOTION_TAG_MAP } from '~utils/colonyMotions';
+import { getMainClasses } from '~utils/css';
+import { formatText } from '~utils/intl';
+
+import styles from './ListItem.css';
+
+const displayName = 'ListItem';
+
+export enum ListItemStatus {
+  NeedAction = 'NeedAction',
+  NeedAttention = 'NeedAttention',
+  Draft = 'Draft',
+  // Default status, does not do anything
+  Defused = 'Defused',
+}
+
+interface FormattedDateProps {
+  parts: {
+    value: string;
+  }[];
+}
+
+const FormattedDate = ({ parts }: FormattedDateProps) => (
+  <>
+    <span className={styles.day}>{parts[2].value}</span>
+    <span>{parts[0].value}</span>
+  </>
+);
+
+interface ListItemProps {
+  /** To be displayed at the end of the list item, e.g. a countdown timer */
+  actions?: ReactNode;
+  /** Avatar to be displayed */
+  avatar?: ReactNode;
+  /** The date the corresponding event was created */
+  createdAt?: string | number | Date;
+  /** Metadata to be displayed beneath the item title. */
+  meta?: ReactNode;
+  /** A click handler for the list item */
+  onClick?: (
+    e: React.MouseEvent<HTMLDivElement> | React.KeyboardEvent<HTMLDivElement>,
+  ) => void;
+  /** An ItemStatus. Styles the list item's 'before' psuedo-element. */
+  status?: ListItemStatus;
+  /** A title */
+  title: Message;
+  /** The tag to be displayed next to the title */
+  tag: MotionState;
+}
+
+const ListItem = ({
+  actions,
+  avatar,
+  createdAt,
+  meta,
+  onClick,
+  status = ListItemStatus.Defused,
+  title,
+  tag,
+}: ListItemProps) => {
+  const tagStyles = MOTION_TAG_MAP[tag];
+
+  return (
+    <li>
+      <div
+        className={getMainClasses({}, styles, {
+          [ListItemStatus[status]]: !!status,
+        })}
+        onClick={onClick}
+        onKeyDown={onClick}
+        role="button"
+        tabIndex={0}
```

Comment:
I don't think the tabIndex-es should be here at all, since we're basically hardcoding all to 0, meaning that keyboard nav is virtually impossible.

Then again, I don't really recall the original reason for why they were placed in here to being with

Reply:
I did try initially without, but the linter doesn't like it: 

> error  Elements with the 'button' interactive role must be focusable  jsx-a11y/interactive-supports-focus